examples/browser-verify/tests/app.rs

```diff
@@ -21,6 +21,6 @@ wasm_bindgen_test_configure!(run_in_browser);
 // This runs a unit test in the browser, so it can use browser APIs.
```

Comment:
I think "it" could be clarified here. 
I tried to suggest a revision to fix this but found myself confused

Reply:
Is this clearer? (I'm not personally find it very confusing, so I'm not sure what needs to be improved.)
```suggestion
// `test_verify` runs a unit test in the browser, so it can use browser APIs.
```